textdescriptives/tests/test_dependency_distance.py

```diff
@@ -0,0 +1,103 @@
+import spacy
+import pytest
+from textdescriptives.components import DependencyDistance
+from .books import *
+import numpy as np
+import ftfy
+
+
+@pytest.fixture(scope="function")
+def nlp():
+    nlp = spacy.load("en_core_web_sm")
```

Comment:
You will need to add en_core_web_sm to your test dependencies (or simply in the requirements.txt)